app/models/status.rb

```diff
@@ -3,7 +3,8 @@ class Status < ActiveRecord::Base
 
   validates :description, presence: true, inclusion: { in: ['Open To Help',
                                                             'Ready To Pair',
-                                                            'In The Zone'] }
+                                                            'In The Zone',
+                                                            'Sleeping'] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.